tests/providers/amazon/aws/operators/test_sagemaker_pipeline.py

```diff
@@ -30,6 +30,9 @@
 )
 from airflow.providers.amazon.aws.triggers.sagemaker import SageMakerPipelineTrigger
 
+if TYPE_CHECKING:
+    from unittest.mock import MagicMock
```

Comment:
The same valid here `from unittest import mock` should also load Magic as well as  `@mock.patch.object(SageMakerHook, "start_pipeline")`